samples/excel/85-preview-apis/tetrominos.yaml

```diff
@@ -655,6 +661,13 @@ template:
                 <div id="sessionPane">
                 </div>
             </section>
+            <p/>
+            <section class="samples ms-font-m" id="focus" hidden="true">
+                <p>Use the <b>Keyboard Focus</b> button to regain task pane focus for the controls if lost.</p>
```

Comment:
The phrasing is maybe _slightly_ strange.  But if you think it's fine, it's fine by me too.